nucypher-core/src/dkg.rs

```diff
@@ -210,22 +518,29 @@ impl<'a> ProtocolObject<'a> for EncryptedThresholdDecryptionRequest {}
 /// A response from Ursula with a derived decryption share.
 #[derive(PartialEq, Eq, Debug, Serialize, Deserialize, Clone)]
 pub struct ThresholdDecryptionResponse {
+    /// The ID of the ritual.
+    pub ritual_id: u16,
```

Comment:
How did we select `u16` for `ritual_id`? Does it have to do with the size of `ritual_id` in the Coordinator contract? I.e. where does the upper bound come from?

Reply:
Ritual ID is `uint32` on Coordinator contract